pages/common/mongo.md

```diff
@@ -0,0 +1,19 @@
+# mongo
+
+> MongoDB shell client.
+
+- Connect to a database:
+
+`mongo {{database}}`
+
+- Connect to a database running on a given host on a given port:
+
+`mongo --host {{host}} --port {{port}} {{database}}`
+
+- Connect to a database with a given username, user will be prompted for password:
+
+`mongo --username {{username}} {{database}} --password`
+
+- Run queries on the database:
```

Comment:
I think the concept of a "javascript expression" as a query is a bit hard to understand to a newbie. So, gave an example instead. And removed "query" from the description.